starter-code/src/viking.js

```diff
@@ -1,11 +1,94 @@
-// Soldier
-function Soldier() {}
+// SOLDIER
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+}
+Soldier.prototype.attack = function(){
+    return this.strength;
+};
+Soldier.prototype.receiveDamage = function(damage){
+    this.health -= damage; 
+};
 
-// Viking
-function Viking() {}
+Soldier.prototype.battleCry = function() {
+    return "Odin Owns You All!"
+}
```

Comment:
Por qué `Soldier` hace `battleCry()`? Os imagináis a todos los Sajones chillando? solo los Vikingos pueden hacer dicha función. Por lo que debería ser `Viking.prototype.battleCry`